components/core/tools/docker-images/clp-env-base-centos-stream-9/Dockerfile

```
@@ -7,6 +7,10 @@ ADD ./tools/scripts/lib_install ./tools/scripts/lib_install
 
 RUN ./tools/scripts/lib_install/centos-stream-9/install-all.sh
 
+# Install uv and set up its cache dir
+RUN pip3 install --no-cache-dir uv==0.8.3
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)